README.md

```diff
@@ -115,7 +115,7 @@ When this variable is set, the run-cnf-suites.sh script will deploy/refresh the
 
 ### Disconnected environment
 In a disconnected environment, only specific versions of images are mirrored to the local repo. For those environments,
-the partner pod image `quay.io/testnetworkfunction/cnf-test-partner` and debug pod image `quay.io/testnetworkfunction/debug-partner` should be mirrored
```

Comment:
We're still using the image for the local-pod-under-test deployment, but I guess that does not make sense beyond CI, right?

Reply:
Yes.